### ISSUE TITLE
SCRF Documentation Strategy

### ISSUE THREAD

**Issue by isss111** [smartcontractresearchforum/discovery#382]
The [Smart Contract Research Forum](https://www.smartcontractresearch.org/) (SCRF) includes a large volume of content outlining programs, processes, guides, and more. Documentation acts as the backbone of SCRF by offering an extensive overview of all its activities.
Documentation houses everything currently going on at SCRF. A comprehensive repository on GitHub [/docs](https://github.com/smartcontractresearchforum/docs) acts as an informative homepage for anyone interested in learning more or currently contributing to SCRF. All of SCRF’s documentation is discoverable to an open online audience.

GDoc Link: https://docs.google.com/document/d/1My6D9wEGNSP3HBSIao4OchRNccLvm-CpN0AQDNYGD9E/edit#heading=h.gjdgxs

**Comment by isss111** [comment]
@marvino3 is this document in its final version?

**Comment by isss111** [comment]
@ralphlombreglia do you mind reviewing this document?

**Comment by ralphlombreglia** [comment]
Hi Ines. Could you please give me the URL to this doc?

**Comment by isss111** [comment]
@ralphlombreglia here it is: https://docs.google.com/document/d/1My6D9wEGNSP3HBSIao4OchRNccLvm-CpN0AQDNYGD9E/edit#heading=h.gjdgxs

**Comment by ralphlombreglia** [comment]
@Inês Santos Silva ***@***.***> This document has been edited.

**Comment by isss111** [comment]
@marvino3 I was looking at this document, and I assessed that this is a strategy doc with a to-do list. I'm not sure if we shouldn't simplify the document, taking the to-do list out and focusing only on strategy. Can you provide some guidance?

**Comment by marvino3** [comment]
I agree, thanks for catching that. Let's simplify and take out the to-do list @isss111.

**Comment by isss111** [comment]
@marvino3 to make sure we don't lose the previous document, I've created a new one: https://docs.google.com/document/d/1uOJfO_LNNHmHd-UgU821al_RJp0nZ-csvSwbiBmgB9s/edit#heading=h.nm8qkst5r0q6

And made it ready for PR: 
https://hackmd.io/@scrf/BJgVyv9Ls
